frontend/components/LoginModal.js

```diff
@@ -0,0 +1,45 @@
+import { useState, useEffect } from 'react'
+import s from 'styled-components'
+
+import Modal from './common/Modal'
```

Comment:
can we make this a named export and export it from `common/index.js` so we can say `import { Modal } from './common'`?

Reply:
Yup!